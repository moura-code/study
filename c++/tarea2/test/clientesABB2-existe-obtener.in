crearClientesABB
crearCliente 10 Artemio Perez 21
agregarAClientesABB
obtenerClienteClientesABB 10
crearCliente 5 Casimira Fernandez 52
agregarAClientesABB
crearCliente 3 Ubaldo Fernandez 19
agregarAClientesABB
crearCliente 2 Esmeralda Lopez 21
agregarAClientesABB
existeEnClientesABB 10
existeEnClientesABB 1
existeEnClientesABB 2
existeEnClientesABB 4
existeEnClientesABB 5
existeEnClientesABB 7
obtenerClienteClientesABB 5
obtenerClienteClientesABB 3
obtenerClienteClientesABB 10
obtenerClienteClientesABB 2
liberarClientesABB
Fin
