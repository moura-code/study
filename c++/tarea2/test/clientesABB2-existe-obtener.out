Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> El clientesABB ha sido creado de forma exitosa.
2> Cliente creado de forma exitosa.
3> Se ha agregado el cliente al clientesABB de forma exitosa.
4> Cliente Artemio Perez
Id: 10
Edad: 21
5> Cliente creado de forma exitosa.
6> Se ha agregado el cliente al clientesABB de forma exitosa.
7> Cliente creado de forma exitosa.
8> Se ha agregado el cliente al clientesABB de forma exitosa.
9> Cliente creado de forma exitosa.
10> Se ha agregado el cliente al clientesABB de forma exitosa.
11> El cliente con id 10 pertenece al clientesABB.
12> El cliente con id 1 NO pertenece al clientesABB.
13> El cliente con id 2 pertenece al clientesABB.
14> El cliente con id 4 NO pertenece al clientesABB.
15> El cliente con id 5 pertenece al clientesABB.
16> El cliente con id 7 NO pertenece al clientesABB.
17> Cliente Casimira Fernandez
Id: 5
Edad: 52
18> Cliente Ubaldo Fernandez
Id: 3
Edad: 19
19> Cliente Artemio Perez
Id: 10
Edad: 21
20> Cliente Esmeralda Lopez
Id: 2
Edad: 21
21> ClientesABB liberado con exito.
22> Fin.
